lib/puma/runner.rb

```diff
@@ -90,26 +90,13 @@ def close_control_listeners
       @control.binder.close_listeners if @control
     end
 
-    # @!attribute [r] ruby_engine
-    def ruby_engine
```

Comment:
This method is public, so we should deprecate it, not remove it.

```ruby
$ ruby -rpuma -rpuma/runner -e 'p Puma::Runner.new(Puma::Launcher.new(Puma::Configuration.new)).ruby_engine'
"ruby 3.2.4-p170"
```

Unless this change should wait for Puma v7?